libexec/estate-change/metadata.yaml

```diff
@@ -29,7 +29,7 @@ help:
   syntax: 'NAME TYPE'
   summary: 'Change the machine type for a node or group'
   # NOTE: If changing the machine type list below also change the
-  # MACHINE_TYPE_MAP in machine-type-definitions.sh.
+  # MACHINE_TYPE_MAP in helpers/aws-machine-type-definitions.sh.
```

Comment:
Can we add a reference to `helpers/azure-...` too please.

Reply:
https://github.com/openflighthpc/flight-action-api/pull/19/commits/c2b1e9f74c61c1d676a82c1c861e68ae2f25a734